babel/index.js

```diff
@@ -453,16 +453,14 @@ BabelClient.prototype.createAnnotation = function createAnnotation(token, data,
     this.debug(JSON.stringify(requestOptions));
 
     request.post(requestOptions, function requestResponse(err, response, body){
-        if(err){
+        if (err) {
             callback(err);
-        } else{
-            if(!this._responseSuccessful(response) || (body.message && body.errors)){
```

Comment:
If we're checking the status code of the response in `_responseSuccessful ` we shouldn't need to check for `body.errors`